CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 Breaking changes:
 
+- [#1645](https://github.com/rails-api/active_model_serializers/pull/1645) Expanded KeyTransform into Transform. Changed :dashed transform to :dash. (@remear)
```

Comment:
Looks like it's still transforming keys or things of that ilk.. what am I missing?